docs/ref/request-response.txt

```diff
@@ -473,20 +473,22 @@ a subclass of dictionary. Exceptions are outlined here:
 
 .. method:: QueryDict.items()
 
-    Just like the standard dictionary ``items()`` method, except this uses the
-    same last-value logic as ``__getitem__()``. For example::
+    Similar to the the standard dictionary ``items()`` method, except this uses the
+    same last-value logic as ``__getitem__()`` and returns an iterable
+    object instead of a view object. For example::
```

Comment:
Do you think if "returns an iterable" is the phrasing that the Python docs use for generators? I would expect something like "yields ..." since there's no "return" statement, but if this is consistent with other places, it's probably okay.